portfolio/src/main/webapp/script.js

```diff
@@ -29,5 +29,32 @@ function addRandomFact() {
 
   // Add it to the page.
   const factContainer = document.getElementById('fact-container');
-  factContainer.innerText = fact;
+  factContainer.innerText = fact; 
+}
+
+/** 
+ * Picks a caption to display depending on the image clicked
+ */
+function displayCaption(img) {
+  const id = img.id;
+  const imageFacts = 
+  {
+	chinesephoenix: `This technically isn't finished yet! A lot of outlining
```

Comment:
Can you use camelcase for the contents of imageFacts?

Reply:
I'm not sure. The attribute names here are meant to match the image id names from the gallery page so I can retrieve them later. I could add camel case to the image ids, but that would conflict with the HTML style guide which says to make everything lowercase. Should I change the image ids anyway?

Reply:
Looks like they have a specific rule for id/class delimiters: https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

Reply:
Ok, I added hyphens!

Reply:
oh good catch Kelly!